CHANGES.rst

```diff
@@ -2,6 +2,14 @@
 Changelog
 =========
 
+v0.9.0 (unreleased)
```

Comment:
Thinking about it, it won't be 0.9.0... Maybe put 0.8.3 as a temporary placeholder and we'll change it to the real patch number when we release the patch in a few days.